Makefile

```diff
@@ -185,8 +185,12 @@ chat2: | build deps librln
 		$(ENV_SCRIPT) nim chat2 $(NIM_PARAMS) $(EXPERIMENTAL_PARAMS) waku.nims
 
 rln-keystore-generator: | build deps librln
-	echo -e $(BUILD_MSG) "build/$@" && \
+ifeq ($(RLN),true)
+		echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim rln_keystore_generator $(NIM_PARAMS) $(EXPERIMENTAL_PARAMS) waku.nims
+else
+		$(error RLN is not true. To build rln-keystore-generator, set RLN=true.)
+endif
```

Comment:
not sure if there's a better way to make RLN default to true while building this target - thoughts @vpavlin?

Reply:
Hey @rymnc !
Maybe we could do sth like the next:
https://gist.github.com/Ivansete-status/fbd0c075658f942b2f9a5c111ae4ef1b#file-makefile-L138
https://gist.github.com/Ivansete-status/fbd0c075658f942b2f9a5c111ae4ef1b#file-makefile-L145
https://gist.github.com/Ivansete-status/fbd0c075658f942b2f9a5c111ae4ef1b#file-makefile-L190

Reply:
oh! good idea, i'll add that
thanks!

Reply:
Addressed in 3a7ebc43